packages/create/src/generators/demoing-storybook-ts/templates/static-scaffold/stories/index.stories.md

```diff
@@ -0,0 +1,48 @@
+```js script
+import { html } from '@open-wc/demoing-storybook';
+import '../out-tsc/<%= tagName %>.js';
```

Comment:
Shouldn't this point to the source file?

Reply:
For the app generator storybook seems to work fine if I link to `../tagName.js` here, but in the component generator I'm getting a missing import for `../tagName.js` in the browser console, not sure why it works for one and not for the other 😕

Reply:
I didn't realize this is in the .md file, then it should be JS.

Reply:
so `import '../out-tsc/<%= tagName %>.js';` is correct here, right?

Reply:
yes

Reply:
mosterd na de maaltijd lars 😂

Reply:
misschien kon je er vannacht niet van slapen